roles/galaxyprojectdotorg.devbox/templates/planemo-machine.sh.j2

```diff
@@ -1 +1,13 @@
-export GALAXY={{ galaxy_root }}
\ No newline at end of file
+export GALAXY={{ galaxy_root }}
+
+if [[ -z $PYTHONPATH ]] ; then
+    export PYTHONPATH={{ dev_user_home }}/miniconda2/lib/python2.7
+else
+    export PYTHONPATH={{ dev_user_home }}/miniconda2/lib/python2.7:$PYTHONPATH
```

Comment:
I don't get why we are doing anything with the Python path at all I mean. Also you are putting conda ahead of system Python on the path - this breaks things. I'd just stick with adding conda to the path, but at the end and not touch Python path (unless there is a documented reason).